BUILDING.md

```diff
@@ -138,13 +140,25 @@ for more information.
 Note that the above requires that `python` resolve to Python 2.6 or 2.7
 and not a newer version.
 
-To run the tests:
+#### Running Tests
+
+To verify the build:
+
+```console
+$ make test-only
```

Comment:
I am wondering if there is anyone out there using the existing `make check` (alias to `make test`) because to me `make check` sounds better than `make test-only` for that purpose.